### PR TITLE
[Android] Add test APKs for shared mode

### PR DESCRIPTION
**Pull request by lincsoon**
Add XWalkSharedShell and XWalkSharedTest for shared mode test, the
package name, test cases, assets of these APKs are just like
XWalkCoreShell and XWalkCoreTest except that they must work with
XWalkRuntimeLib.

### CODE REVIEW

**Comment by rakuco** [comment]
Please remember to file JIRA tickets for those APKs to start being shipped with each build, as well as added to the bot test list.

**Comment by lincsoon** [comment]
@rakuco These new APKs don't need to be shipped nor to be included into the bot because they are the exactly same with XWalkCoreShell and XWalkCoreTest. They are only for compatibility test of shared mode. Before every beta release, QA will run XWalkSharedTest on different versions of XWalkRuntimeLib manually.

**Comment by lincsoon** [comment]
@rakuco Could you give a LGTM so that I can merge this.

File: xwalk_android_tests.gypi [hunk 1]
```diff
@@ -138,6 +182,7 @@
         'xwalk_core_shell_apk_java',
         '../tools/android/md5sum/md5sum.gyp:md5sum',
         '../tools/android/forwarder2/forwarder.gyp:forwarder2',
+        'xwalk_xwview_test_assets',
```
Comment (wuhengzhi) [inline xwalk_android_tests.gypi]: Reason for adding this?

**Comment by lincsoon** [comment]
@rakuco

**Comment by rakuco** [comment]
lgtm
I still think you need to file a ticket to get this into the bot/release build workflow, otherwise we won't even know if it's minimally running, and if it's not part of the zip files QA will need to build the APKs on their own.

**Comment by lincsoon** [comment]
Thanks for the reply. I'll file a ticket to ship this.

**Pull request merged**